src/emu/debug/debughlp.cpp

```diff
@@ -78,6 +93,7 @@ static const help_item static_help_list[] =
 		"Type help <command> for further details on each command\n"
 		"\n"
 		"  help [<topic>] -- get help on a particular topic\n"
+		"  html -- print all debugger help to an html file\n"
```

Comment:
`html` isn't a good name, it doesn't hint that it does something with help.  Also, where's the placeholder for the filename.

Reply:
How should I call the command? html should be mentioned somewhere, it's the file type I output. Probably later on, I'll come up with a way to output a wiki doc file, but the amount of file types will be very narrow.